plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -569,7 +569,7 @@ public void testMergeSimpleSelectPartitioned()
 
         assertUpdate(format("INSERT INTO %s (customer, purchases, address) VALUES ('Aaron', 5, 'Antioch'), ('Bill', 7, 'Buena'), ('Carol', 3, 'Cambridge'), ('Dave', 11, 'Devon')", targetTable), 4);
 
-        assertUpdate(format("CREATE TABLE %s (customer VARCHAR, purchases INT, address VARCHAR) WITH (location = 's3://%s/%s')", sourceTable, bucketName, sourceTable));
+        assertUpdate(format("CREATE TABLE %s (customer VARCHAR, purchases INT, address VARCHAR) WITH (location = 's3://%s/%s/')", sourceTable, bucketName, sourceTable));
```

Comment:
Instead of modifying an existing test to do double duty, I might add another simple test (`testTableLocationTrailingSlash`, I guess) that just creates two tables (one with a slash and one without), inserts a value into them, selects it, and drops them.

If someone else think's that's too much for this small change, then there should at least be a comment on the two `create table` lines in this test pointing out that they are intentionally different for testing purposes.